plasma_framework/contracts/src/transactions/outputs/PaymentOutputModel.sol

```diff
@@ -21,15 +21,15 @@ library PaymentOutputModel {
      *       This should not and cannot be handled here.
      */
     function owner(Output memory _output) internal pure returns (address payable) {
-        return AddressPayable.convert(address(uint256(_output.outputGuard)));
+        return AddressPayable.convert(address(uint160(_output.outputGuard)));
     }
 
     function decode(RLP.RLPItem memory encoded) internal pure returns (Output memory) {
         RLP.RLPItem[] memory rlpEncoded = encoded.toList();
         require(rlpEncoded.length == 3, "Invalid output encoding");
 
         Output memory output = Output({
-            outputGuard: rlpEncoded[0].toBytes32(),
+            outputGuard: bytes20(rlpEncoded[0].toAddress()),
```

Comment:
do you think it worth to introduce a `toBytes20` function in RLP?

Reply:
done